frontend/src/bundles/common/pages/not-found/not-found.tsx

```diff
@@ -0,0 +1,45 @@
+import { useCallback } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import {
+    Button,
+    ButtonSize,
+    ButtonVariant,
+} from '../../components/components.js';
+import {
+    NotFoundIcon,
+    NotFoundIconColor,
+    NotFoundIconSize,
+} from '../../components/icons/icons.js';
+import { AppRoute } from '../../enums/app-route.enum.js';
+
+const NotFound: React.FC = () => {
+    const navigate = useNavigate();
+
+    const redirectToHomepage = useCallback((): void => {
+        navigate(AppRoute.ROOT);
+    }, [navigate]);
+
+    return (
+        <div className="bg-lm-black-200 flex flex-1 items-center justify-center">
+            <div className="text-center">
+                <NotFoundIcon
+                    color={NotFoundIconColor.PRIMARY}
+                    size={NotFoundIconSize.EXTRA_LARGE}
+                />
+                <h1 className=" mt-5 text-4xl text-white">
+                    Oops, this page not found
```

Comment:
I think it is better to use either the short form something like _Page Not Found_ or a full sentence with a verb like _Oops, this page **is/was** not found_